account_financial_report_qweb/static/src/less/report.less

```diff
@@ -103,4 +103,5 @@
     margin-right: auto;
     margin-left: auto;
     font-family: Helvetica, Arial;
+    padding: 20px;
```

Comment:
But this padding is still changed...

Reply:
because w/ less file background compiled to white color, and this padding we need

Reply:
@jbeficent what do you think about these changes? I still don't like the white background and the unneeded padding